syft/frameworks/torch/tensors/interpreters/native.py

```diff
@@ -992,19 +992,57 @@ def torch_type(self):
         else:
             return self.child.torch_type()
 
-    def encrypt(self, public_key):
-        """This method will encrypt each value in the tensor using Paillier
-        homomorphic encryption.
+    def encrypt(self, encryption_method="mpc", **kwargs):
+        """
+        This method will encrypt each value in the tensor using Multi Party
+        Computation (default) or Paillier Homomorphic Encryption
 
         Args:
-            *public_key a public key created using
-                syft.frameworks.torch.he.paillier.keygen()
+            encryption_method (str): Currently supports 'mpc' for Multi Party
+                Computation and 'paillier' for Paillier Homomorphic Encryption
+            **kwargs:
+                With Respect to MPC accepts:
+                    workers (list): Parties involved in the sharing of the Tensor
+                    crypto_provider (syft.VirtualWorker): Worker responsible for the
+                        generation of the random numbers for encryption
+                    args_fix_prec (dict): To be parsed as a kwarg for the .fix_prec() method
```

Comment:
Could you also allow fix_precision kwargs to be given alongside the share ones (workers & crypto_provider)? Would be simpler for the end user :)

Reply:
(I think we would provide kwargs and not args for fix_precision)